distributed/worker.py

```diff
@@ -440,6 +440,8 @@ def __init__(self, scheduler_ip=None, scheduler_port=None,
         self.service_specs = services or {}
         self.metrics = dict(metrics) if metrics else {}
 
+        self.low_level_profiler = low_level_profiler
```

Comment:
We might want to use configuration here as well.  

```python
dask.config.get('distributed.worker.low-level-profiler')
```

you would also have to add this to the `distributed.yaml` file in this repository.

Reply:
Ah, that sounds good. Should I remove then the parameter from `worker.__init__()` in favor of the configuration?